Packs/FreshworksFreshservice/Integrations/FreshworksFreshservice/FreshworksFreshservice.py

```diff
@@ -0,0 +1,3630 @@
+        return arg_template[0]
+
+    return arg_template
+
+
+def update_custom_fields(args: Dict[str, Any]) -> Dict[Any, Any] | None:
+    """ Update custom_fields argument to match Freshservice template.
+
+    Args:
+        args (Dict[str, Any]): command arguments from XSOAR.
+
+    Returns:
+        Optional[Dict[Any, Any]]: Updated field.
+    """
+    updated_custom_fields = None
+    if custom_fields := args.get('custom_fields'):
+        custom_fields = custom_fields.split('=')
+        updated_custom_fields = {custom_fields[0]: custom_fields[1]}
+    return updated_custom_fields
+
+
+def validate_mandatory_ticket_requester_fields(
+    entity_name: str,
+    args: Dict[str, Any],
+    entity_id_value: str = None,
+):
+    """ Validate user specified one of the following:
+        requester_id, phone, email
+        when create any type of ticket.
+
+    Args:
+        entity_name (str): The command entity name.
+        args (Dict[str, Any]): command arguments from XSOAR.
+        entity_id_value (str | None, optional): Ticket ID. Defaults to None.
+
+    Raises:
+        ValueError: In case user don't specified the required fields.
+    """
+    if all([
+            entity_name != 'release', not entity_id_value, not any([
+                args.get('requester_id'),
+                args.get('phone'),
+                args.get('email')
+            ])
+    ]):
+        raise ValueError(
+            'One of the following is mandatory: requester_id, phone, email')
+
+
+def get_files_to_attach(args: Dict[str, Any], ) -> List[Tuple]:
+    """ Get array of files to attach in Freshservice.
+
+    Args:
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        List[Tuple]: List of files.
+    """
+    return [get_file(file) for file in argToList(args.get('attachments'))]
+
+
+def convert_response_properties(
+    updated_response: List[dict] | Dict[str, Any],
+    predefined_values: Dict[str, Dict[str, int]],
+) -> List[dict] | Dict[str, Any]:
+    """ Convert command properties from number to string for
+        the XSOAR output.
+
+    Args:
+        updated_response (List[dict]): Freshservice response.
+        predefined_values (Dict[str, Dict[str, int]]): Predefined values for Freshservice.
+
+    Returns:
+        List[dict]: Updated response.
+    """
+    if not updated_response:
+        return updated_response
+
+    if isinstance(updated_response, dict):
+        updated_response = [updated_response]
+
+    response_keys = updated_response[0].keys()
+
+    for item in updated_response:
+        for response_key in response_keys:
+            if item.get(
+                    response_key) and response_key in predefined_values.keys():
+
+                property_values = reverse_dict(predefined_values[response_key])
+                item[response_key] = property_values.get(item[response_key])
+
+    return updated_response
+
+
+def reverse_dict(dict_: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
We don't need a function for one line

Reply:
But why would we want to repeat a logic multiple times? Also, it will be easier to read and maintain if any changes are needed.